### PR TITLE
hw-26-2

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

**Review by vladkuzya — APPROVED** [review]

File: src/main/java/com/internet/shop/controller/order/GetUserOrdersController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class GetUserOrdersController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final OrderService orderService
+            = (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Order> allOrders = orderService.getUserOrders(USER_ID);
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controller/order/GetUserOrdersController.java]:
```suggestion
        List<Order> userOrders = orderService.getUserOrders(USER_ID);
```

File: src/main/java/com/internet/shop/controller/order/CreateOrderController.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/orders/create")
+public class CreateOrderController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controller/order/CreateOrderController.java]:
```suggestion
public class CompleteOrderController extends HttpServlet {
```
it's better to keep naming consistent

**Review by RomanDubovskyi — COMMENTED** [review]
great 👍 , just a minor comment

File: src/main/java/com/internet/shop/controller/order/GetUserOrdersController.java (hunk 1 above)
Comment (Rommelua) [reply src/main/java/com/internet/shop/controller/order/GetUserOrdersController.java]: done

File: src/main/java/com/internet/shop/controller/order/CreateOrderController.java (hunk 2 above)
Comment (Rommelua) [reply src/main/java/com/internet/shop/controller/order/CreateOrderController.java]: done

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**